style.css

```diff
@@ -1,138 +1,149 @@
-@import url('https://fonts.googleapis.com/css2?family=Inter:wght@500;700&display=swap');
+html,
+body {
+  max-width: 100%;
+  overflow-x: hidden;
+}
```

Comment:
- [ ] Great work! You are making progress! :+1: It might help to watch the attached video to gain some insights on how to fix this seeming issue of responsiveness. Implementing previous reviewer's suggestion is pivotal to completing this milestone. Here is the [video recording](https://www.awesomescreenshot.com/video/15319090?key=ac6e519bdcccb000c47f251e4a92e2a1) for tips.